src/main/webapp/js/stats-page-loader.js

```diff
@@ -0,0 +1,29 @@
+const urlParams = new URLSearchParams(window.location.search);
+const parameterUsername = urlParams.get('stats');
+
+/** Fetches stats and add them to the page. */
+function fetchStats(){
```

Comment:
Please add space before {.
e.g: 
function fetchStats() {

Please apply this comment to all the methods in this file and other files (if applicable).

Reply:
Addressed